docs/quick/k8s/.gitignore

```diff
@@ -0,0 +1,2 @@
+.cache/
+kustomize/
```

Comment:
Why do you need this `.gitignore` file?

Reply:
When you execute `kfctl apply -V -f ...`, it will generate this two folder.